src/AutoRest.CSharp/Output/Models/LowLevelRestClient.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using AutoRest.CSharp.Generation.Types;
+using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Output.Models.Requests;
+using AutoRest.CSharp.Output.Models.Shared;
+using AutoRest.CSharp.Output.Models.Types;
+
+namespace AutoRest.CSharp.Output.Models
+{
+    internal class LowLevelRestClient : ClientBase
+    {
+        private readonly OperationGroup _operationGroup;
+        private readonly BuildContext<LowLevelOutputLibrary> _context;
+        private RestClientBuilder<LowLevelOutputLibrary> _builder;
+
+        private Dictionary<ServiceRequest, RestClientMethod>? _requestMethods;
+        private RestClientMethod[]? _allMethods;
+
+        protected override string DefaultAccessibility { get; } = "public";
+
+        public LowLevelRestClient(OperationGroup operationGroup, BuildContext<LowLevelOutputLibrary> context) : base(context)
+        {
+            _operationGroup = operationGroup;
+            _context = context;
+            _builder = new RestClientBuilder<LowLevelOutputLibrary> (operationGroup, context, FilterMethodParameters);
+
+            Parameters = _builder.GetOrderedParameters ();
+            ClientPrefix = GetClientPrefix(operationGroup.Language.Default.Name, context);
+            DefaultName = ClientPrefix + ClientSuffix;
+            Description = "";
+        }
+
+        public Parameter[] Parameters { get; }
+        public string Description { get; }
+        public RestClientMethod[] Methods => _allMethods ??= BuildAllMethods().ToArray();
```

Comment:
We don't generate any special methods (LRO, next page, etc) for LLC, do we need both GetAll and GetNormal methods?

Reply:
Methods property uses BuildAllMethods()
BuildAllMethods() uses GetOperationMethod()
GetOperationMethod() uses EnsureNormalMethods()

Reply:
Can we merge everything into BuildAllMethods?